csl.rnc

```diff
@@ -717,6 +718,71 @@ div {
        attribute variable { variables.standard },
        [ a:defaultValue = "long" ] attribute form { "short" | "long" }?)
 }
+[ a:documentation [ xhtml:h2 [ "cs:title Rendering Element" ] ] ]
+div {
+  ## Use to render titles if "title-main" and "title-sub" need different formatting
```

Comment:
Add `## or to omit either "title-main" or "title-sub"`

Reply:
Thanks.